### PR TITLE
Word/character replacement and addition of code markup

### PR DESCRIPTION
**Pull request by konrad**

### CODE REVIEW

File: 02-create.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ $ mkdir planets
 $ cd planets
 ~~~
 
-and tell Git to make it a [repository](reference.html#repository)&mdash;a place where
+and tell Git to make it a [repository](reference.html#repository) &ndash; a place where
```
Comment (daisieh) [inline 02-create.md]: Actually, neither of these is correct. Can you make it into a comma?

Comment (gvwilson) [inline 02-create.md]: Why isn't the em-dash correct?

Comment (konrad) [inline 02-create.md]: My mistake - em-dash seems to be correct (http://www.thepunctuationguide.com/em-dash.html).

Comment (daisieh) [inline 02-create.md]: It's not that you *couldn't* use an em-dash there, it just seems unnecessary. A comma would suffice and be accurate. There is no need for emphasis of the appositive phrase.

Comment (daisieh) [inline 02-create.md]: But it's up to you: either revert this back to the em-dash or use a comma, please.

**Comment by iglpdc** [comment]
Thanks @konrad, I like it. However, I'm not sure if adding the configuration for emacs and vim is useful here. We're trying to give instructions only for the editors we advice to install for our workshops (the list is already too long for me.) Furthermore, emacs and vim are advanced editors that our learners don't typically use.

**Comment by konrad** [comment]
Thanks for the feedback @iglpdc. I thought that people who might be familiar with Emacs or Vim might be not necessarily the audience of the course. Stil I gave it a try. I will remove them.

**Comment by iglpdc** [comment]
Thanks, @konrad. I'm merging it.

**Comment by konrad** [comment]
Great! Many thanks, @iglpdc.

**Pull request merged**